pkg/cluster/internal/providers/podman/provider.go

```diff
@@ -60,10 +60,8 @@ func (p *Provider) Provision(status *cli.Status, cfg *config.Cluster) (err error
 		return err
 	}
 
-	// kind doesn't work with podman rootless, surface an error
 	if os.Geteuid() != 0 {
-		p.logger.Errorf("podman provider does not work properly in rootless mode")
```

Comment:
I don't think we should stop failing until this works, actually, this was previous state but it was confusing for users